src/screens/GroupInfo/elements/ItemLink.js

```diff
@@ -3,6 +3,7 @@ import {Image, Linking, TouchableOpacity, StyleSheet, Text, View} from 'react-na
 import {colors} from '../../../utils/colors';
 import {fonts} from '../../../utils/fonts';
 import {trimString} from '../../../utils/string/TrimString';
+import {COLORS} from '../../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `COLORS` from the theme file is a good change. It replaces hardcoded color values with dynamic ones, improving modularity and consistency.
```diff
+ import {COLORS} from '../../../utils/theme';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->